### PR TITLE
Service command prefix

### PR DESCRIPTION
**Pull request by dkovacevic**

### CODE REVIEW

File: backend/src/main/java/com/wire/bots/roman/MessageHandler.java [hunk 1]
```diff
@@ -430,6 +435,24 @@ private boolean send(OutgoingMessage message) {
...
+        if (message.type.equalsIgnoreCase("conversation.bot_request"))
+            return false;
+
+        if (provider.commandPrefix == null)
+            return false;
+
+        if (provider.commandPrefix.equalsIgnoreCase("***"))
+            return true;
+
+        if (!message.type.equalsIgnoreCase("text"))
```
Comment (LukasForst) [inline backend/src/main/java/com/wire/bots/roman/MessageHandler.java:448]: so you want to ignore all assets for example? is that correct?

File: backend/src/main/java/com/wire/bots/roman/MessageHandler.java [hunk 2]
```diff
@@ -430,6 +435,24 @@ private boolean send(OutgoingMessage message) {
...
+            return true;
+
+        final String text = message.text.data;
+
+        return !text.startsWith(provider.commandPrefix);
```
Comment (LukasForst) [inline backend/src/main/java/com/wire/bots/roman/MessageHandler.java:453]: if text does not correspond to command prefix, it's ignored, is that correct? shouldn't it be other way around? that if it starts with command prefix, it's ignored?

**Review by LukasForst — CHANGES_REQUESTED** [review]
few questions

File: backend/src/main/java/com/wire/bots/roman/MessageHandler.java (hunk 1 above)
Comment (dkovacevic) [reply backend/src/main/java/com/wire/bots/roman/MessageHandler.java:448]: if this is a command prefix bot then yes.. only commands are sent to the bot

Comment (LukasForst) [reply backend/src/main/java/com/wire/bots/roman/MessageHandler.java:448]: ok, fair enough

File: backend/src/main/java/com/wire/bots/roman/MessageHandler.java (hunk 2 above)
Comment (dkovacevic) [reply backend/src/main/java/com/wire/bots/roman/MessageHandler.java:453]: Yes. Unless the text does not start with that prefix the whole message is ignored (not forwarded to the bot)

Comment (LukasForst) [reply backend/src/main/java/com/wire/bots/roman/MessageHandler.java:453]: aaaah, makes sense now, sorry, I misinterpreted that!

**Review by LukasForst — APPROVED** [review]

**Comment by LukasForst** [comment]
approved!

**Pull request merged**